pkgs/by-name/dw/dwlmsg/README.md

```diff
@@ -0,0 +1,16 @@
+This package depends on DWL being built with the IPC patch found at https://codeberg.org/dwl/dwl-patches/raw/commit/d235f0f88ed069eca234da5a544fb1c6e19f1d33/patches/ipc/ipc.patch
```

Comment:
if this package does not build without patch shouldn't this be part of the package.nix 

see: pkgs/applications/audio/jackmix/default.nix as example or grep for fetchpatch for more examples